internal/internal_versioning_client.go

```diff
@@ -126,11 +128,51 @@ type (
 		WorkerVersionCapabilities *WorkerVersionCapabilities
 	}
 
+	// TaskQueueStats contains statistics about task queue backlog and activity.
+	//
+	// For workflow task queue type, this result is partial because tasks sent to sticky queues are not included. Read
+	// comments above each metric to understand the impact of sticky queue exclusion on that metric accuracy.
+	TaskQueueStats struct {
+		// The approximate number of tasks backlogged in this task queue. May count expired tasks but eventually
+		// converges to the right value. Can be relied upon for scaling decisions.
+		//
+		// Special note for workflow task queue type: this metric does not count sticky queue tasks. However, because
```

Comment:
Does the sticky task queue here get its own `TaskQueueStats`?

Reply:
There is no plan for supporting user-facing API for TaskQueueStats for sticky queues. There will be a tdbg DescribeTaskQueuePartition which will support sticky queues. Also, the backlog hint will be still sent in poll response.

@Quinn-With-Two-Ns Please lmk if there is objections/concerns.

Reply:
Talked offline, there may be some cases where not knowing stats for sticky task queue may make more difficult, but this is a know/accepted limitation of the API